mockld/sdk_data.go

```diff
@@ -166,27 +166,46 @@ func normalizeSegment(key string, data json.RawMessage) (json.RawMessage, error)
 }
 
 type ServerSDKDataBuilder struct {
-	flags    map[string]json.RawMessage
-	segments map[string]json.RawMessage
+	flags           map[string]json.RawMessage
+	segments        map[string]json.RawMessage
+	configOverrides map[string]json.RawMessage
+	metrics         map[string]json.RawMessage
 }
 
 func NewServerSDKDataBuilder() *ServerSDKDataBuilder {
 	return &ServerSDKDataBuilder{
-		flags:    make(map[string]json.RawMessage),
-		segments: make(map[string]json.RawMessage),
+		flags:           make(map[string]json.RawMessage),
+		segments:        make(map[string]json.RawMessage),
+		configOverrides: make(map[string]json.RawMessage),
+		metrics:         make(map[string]json.RawMessage),
 	}
 }
 
 func (b *ServerSDKDataBuilder) Build() ServerSDKData {
 	flags := make(map[string]json.RawMessage)
 	segments := make(map[string]json.RawMessage)
+	configOverrides := make(map[string]json.RawMessage)
+	metrics := make(map[string]json.RawMessage)
+
```

Comment:
I wonder.. can we replace all of these for loops with [`map.Clone`](https://pkg.go.dev/golang.org/x/exp/maps#Clone) now?

Reply:
Yep, that seems to work just fine.